external/kafka-0-10-sql/src/main/scala/org/apache/spark/sql/kafka010/CachedKafkaConsumer.scala

```diff
@@ -45,9 +46,6 @@ private[kafka010] case class CachedKafkaConsumer private(
 
   private var consumer = createConsumer
 
-  /** indicates whether this consumer is in use or not */
-  private var inuse = true
```

Comment:
Don't we need to track this anymore? Will we evict a consumer in use?

Reply:
Now that we have moved to using object pools, this tracking is no longer required.